base/docs/helpdb/Base.jl

```diff
@@ -784,12 +784,12 @@ Equivalent to `stat(file).mode`
 filemode
 
 """
-    print_joined(io, items, delim, [last])
+    join(io, items, delim, [last])
```

Comment:
I wasn't sure if the signature-matching supported that, since it doesn't allow the brackets on the line below specifying which method is documented.